Sources/FirebaseFirestore/FirestoreSource+Swift.swift

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+@_exported
+import firebase
+
+public typealias FirestoreSource = firebase.firestore.Source
+
+extension FirestoreSource: @unchecked Sendable {}
```

Comment:
Is the underlying type actually sendable or is this to help suppress a warning?

Reply:
This is help suppress potential warnings. The underlying type is just an enum (Int32) -- therefore it is sendable. It is just plain old data with value semantics. Unfortunately Swift needs you to still indicate that such types are sendable even though they obviously are.